airbyte-server/src/main/java/io/airbyte/server/Application.java

```diff
@@ -9,7 +9,12 @@
 public class Application {
 
   public static void main(final String[] args) {
-    Micronaut.run(Application.class, args);
+    Micronaut
+        .build(args)
+        // Lazy initialization can make the first requests to be slow
+        .eagerInitSingletons(true)
```

Comment:
This may be a bit dangerous, as it will initialize all singletons, including those in transitive dependencies.  It's fine for now, but if we start to see issues, we may want to use the [Context annotation](https://docs.micronaut.io/latest/api/io/micronaut/context/annotation/Context.html) to explicitly mark beans as eager.  I'm not sure if that would work on a `@Controller` annotated class, so we may have to switch back to the JAX-RS annotations + `@Context`.

Reply:
Done